src/D2L.Bmx/LoginHandler.cs

```diff
@@ -0,0 +1,13 @@
+namespace D2L.Bmx;
+
+internal class LoginHandler(
+	OktaAuthenticator oktaAuth
+) {
+	public async Task HandleAsync(
+		string? org,
+		string? user
+	) {
+		await oktaAuth.AuthenticateAsync( org, user, nonInteractive: false );
+		Console.WriteLine( "Successfully logged in." );
```

Comment:
If there's an error in okta auth it will show the error message instead of ever reaching this line